### PR TITLE
[docs][cloud-provider-vsphere] Fix forgotten content

### PR DESCRIPTION
**Pull request by asviel**
## Description

Fix forgotten in https://github.com/deckhouse/deckhouse/pull/5743 content

## Why do we need it, and what problem does it solve?
Just housekeeping.

## Why do we need it in the patch release (if we do)?
We don't.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: docs
type: chore
summary: Fix forgotten content (list of permissions for vSphere).
impact_level: low
```

### CODE REVIEW

File: ee/candi/cloud-providers/vsphere/docs/ENVIRONMENT_RU.md [hunk 1]
```diff
@@ -97,7 +95,7 @@ govc role.create deckhouse \
    $(govc role.ls Admin | grep -F -e 'Folder.' -e 'InventoryService.' -e 'Resource.' -e 'VirtualMachine.')
 ```
 
-Назначьте пользователю роль на объекте `vCenter`:
+В соответствии созданной ролью назначьте пользователю права на объект `vCenter`, где нужно развернуть кластер Kubernetes:
```
Comment (z9r5) [inline ee/candi/cloud-providers/vsphere/docs/ENVIRONMENT_RU.md:98]:
```suggestion
В соответствии с созданной ролью назначьте пользователю права на объект `vCenter`, где нужно развернуть кластер Kubernetes:
```

File: ee/candi/cloud-providers/vsphere/docs/ENVIRONMENT_RU.md [hunk 2]
```diff
@@ -97,7 +95,7 @@ govc role.create deckhouse \
    $(govc role.ls Admin | grep -F -e 'Folder.' -e 'InventoryService.' -e 'Resource.' -e 'VirtualMachine.')
 ```
 
-Назначьте пользователю роль на объекте `vCenter`:
+В соответствии с созданной ролью назначьте пользователю права на объект `vCenter`, где нужно развернуть кластер Kubernetes:
```
Comment (z9r5) [inline ee/candi/cloud-providers/vsphere/docs/ENVIRONMENT_RU.md:98]:
```suggestion
Назначьте пользователю права на объект `vCenter`, где нужно развернуть кластер Kubernetes:
```

File: ee/candi/cloud-providers/vsphere/docs/ENVIRONMENT.md [hunk 3]
```diff
@@ -96,7 +96,7 @@ govc role.create deckhouse \
    $(govc role.ls Admin | grep -F -e 'Folder.' -e 'InventoryService.' -e 'Resource.' -e 'VirtualMachine.')
 ```
 
-Assign the role to a user on the `vCenter` object:
+In accordance with the created role, assign rights to a user to use the `vCenter` object where you want to deploy the Kubernetes cluster:
```
Comment (z9r5) [inline ee/candi/cloud-providers/vsphere/docs/ENVIRONMENT.md:99]:
```suggestion
Assign rights to a user to use the `vCenter` object where you want to deploy the Kubernetes cluster:
```

File: ee/candi/cloud-providers/vsphere/docs/ENVIRONMENT.md [hunk 4]
```diff
@@ -85,9 +85,9 @@ We've intentionally skipped User creation since there are many ways to authentic
 This all-encompassing Role should be enough for all Deckhouse components. If you need a more granular Role, please contact your Deckhouse support.
 {% endalert %}
 
-You have to create a role with the following list of permissions and assign it to `vCenter`.
+You have to create a role with the following list of permissions and assign .
```
Comment (z9r5) [inline ee/candi/cloud-providers/vsphere/docs/ENVIRONMENT.md:89]:
```suggestion
```

File: docs/site/_includes/getting_started/vsphere/STEP_ENV.md [hunk 5]
```diff
@@ -105,7 +103,7 @@ govc role.create deckhouse \
 ```
 {% endsnippetcut %}
 
-Assign the role to a user on the `vCenter` object:
+Assign the user the role on the `vCenter` object:
```
Comment (kirkonru) [inline docs/site/_includes/getting_started/vsphere/STEP_ENV.md:106]:
```suggestion
Assign the role to a user on the `vCenter` object:
```

File: ee/candi/cloud-providers/vsphere/docs/ENVIRONMENT.md [hunk 6]
```diff
@@ -96,7 +94,7 @@ govc role.create deckhouse \
    $(govc role.ls Admin | grep -F -e 'Folder.' -e 'InventoryService.' -e 'Resource.' -e 'VirtualMachine.')
 ```
 
-Assign the role to a user on the `vCenter` object:
+Assign the user the role on the `vCenter` object:
```
Comment (kirkonru) [inline ee/candi/cloud-providers/vsphere/docs/ENVIRONMENT.md:97]:
```suggestion
Assign the role to a user on the `vCenter` object:
```

File: ee/modules/030-cloud-provider-vsphere/docs/CONFIGURATION_RU.md [hunk 7]
```diff
@@ -51,11 +51,23 @@ force_searchable: true
 
 ## Список привилегий для использования модуля
 
+> О том как создать и назначить роль пользователю, читайте [в документации.](environment.html#создание-и-назначение-роли)
```
Comment (kirkonru) [inline ee/modules/030-cloud-provider-vsphere/docs/CONFIGURATION_RU.md:54]:
```suggestion
> О том, как создать и назначить роль пользователю, читайте [в документации](environment.html#создание-и-назначение-роли).
```

**Review by kirkonru — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [7341](https://github.com/deckhouse/deckhouse/pull/7341) to the branch [release-1.57](https://github.com/deckhouse/deckhouse/tree/release-1.57) successful!

**Pull request merged**